.github/renovate.json

```diff
@@ -1,22 +1,37 @@
 {
-    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-    "extends": [
-      "config:base",
-      "schedule:monthly",
-      ":maintainLockFilesMonthly",
-      ":preserveSemverRanges",
-      ":rebaseStalePrs",
-      ":disableDependencyDashboard"
-    ],
-    "enabledManagers": [
-      "nuget"
-    ],
-    "packageRules": [
-      {
-        "matchManagers": ["nuget"],
-        "groupName": "Nuget updates",
-        "groupSlug": "nuget",
-        "separateMajorMinor": false
-      }
-    ]
-  }
\ No newline at end of file
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":combinePatchMinorReleases",
+    ":dependencyDashboard",
+    ":maintainLockFilesWeekly",
+    ":pinAllExceptPeerDependencies",
+    ":prConcurrentLimit10",
+    ":rebaseStalePrs",
+    "schedule:weekends",
+    ":separateMajorReleases"
+  ],
+  "enabledManagers": ["cargo", "github-actions", "npm", "nuget"],
```

Comment:
💭 I am guessing not all of these are used, but this should be fine to leave as-is.